cores/esp8266/cont_util.cpp

```diff
@@ -73,7 +73,7 @@ void cont_repaint_stack(cont_t *cont)
     register uint32_t *sp asm("a1");
     // Ensure 64 bytes adjacent to the current SP don't get touched to endure
     // we don't accidentally trounce over locals or IRQ temps.
-    uint32_t sp_safe = CONT_STACKSIZE/4 - ((sp - &cont->stack[0] - 64)/4);
+    uint32_t sp_safe = CONT_STACKSIZE/4 - (sp - &cont->stack[0]) - 64/4;
```

Comment:
Please check the sign of the arithmetic

Reply:
Yeah, mybad.  Just rewrote to be saner and have a quick test:
````
void __attribute__((noinline)) bigstack() {
  char buff[1024];
  memset(buff, 0xee, 1024);
  int sum=0;
  for (int i=0; i<1024; i++) {
    sum += buff[i];
  }
  Serial.printf("sum: %d\n", sum);
}

void setup() {
  Serial.begin(115200);
  Serial.printf("StackFree 1: %d\n", ESP.getFreeContStack());
  ESP.resetFreeContStack();
  Serial.printf("StackFree 2: %d\n", ESP.getFreeContStack());
  bigstack();
  Serial.printf("StackFree 3: %d\n", ESP.getFreeContStack());
  ESP.resetFreeContStack();
  Serial.printf("StackFree 4: %d\n", ESP.getFreeContStack());
}

void loop() {}
````
gives
````
StackFree 1: 3280
StackFree 2: 3988
sum: 243712
StackFree 3: 2256
StackFree 4: 3988
````